src/components/ReviewComponents/ReviewComponentsPage.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  Card,
+  CardExpandableContent,
+  CardHeader,
+  Text,
+  TextVariants,
+  Flex,
+  FlexItem,
+} from '@patternfly/react-core';
+import * as React from 'react';
+import { createApplication, createComponent } from '../..//utils/create-utils';
+import { FormFooter } from '../../shared';
+import { useFormValues } from '../form-fields/form-context';
+import { Page } from '../Page';
+import { useWizardContext } from '../Wizard/Wizard';
+import './ReviewComponentsPage.scss';
+
+export const ReviewComponentsPage: React.FC = () => {
+  const wizardContext = useWizardContext();
+  const [formState] = useFormValues();
+  const [isExpanded, setIsExpanded] = React.useState<boolean>(false);
+
+  const handleSubmit = React.useCallback(() => {
+    createApplication(formState.application).then((applicationData) => {
+      // eslint-disable-next-line no-console
+      console.log('###############- Application created', applicationData);
+      createComponent(
+        { name: formState.component.name, gitRepo: formState.component.git.remotes.origin },
+        applicationData?.metadata?.name,
+      ).then((componentData) => {
+        // eslint-disable-next-line no-console
+        console.log('###############- Component created', componentData);
+      });
+    });
+  }, [formState]);
+
+  return (
+    <Page
+      breadcrumbs={[
+        { path: '#', name: 'Applications' },
```

Comment:
Do all pages have the same breadcrumb?

Reply:
yes, for now all pages have the same breadcrumbs.